util/codec/codec.go

```diff
@@ -172,3 +172,145 @@ func DecodeKey(b []byte) ([]interface{}, error) {
 
 	return values, nil
 }
+
+// EncodeValue encodes args to a byte slice which can be decoded by DecodeValue later.
+// It does not guarantee the order for comparison.
+func EncodeValue(args ...interface{}) ([]byte, error) {
```

Comment:
If you use varint/uvarint for number, I think we should add a new flag. 
btw, don't forget adding a comment for these functions that they are not memory comparable.